vault/resource_aws_secret_backend.go

```diff
@@ -99,6 +100,23 @@ func awsSecretBackendResource() *schema.Resource {
 				Optional:    true,
 				Description: "Specifies a custom HTTP STS endpoint to use.",
 			},
+			consts.FieldSTSRegion: {
```

Comment:
This field predates the linked PR but wasn't included in TFVP yet, so i added it while I was here.

(See: https://github.com/hashicorp/vault/pull/29051/files#diff-11906daf04096f16c364abff4761df97c3b879915152cee7f14a03e1f74dc32eL51)

Reply:
Nice! 
Do we know in what release this was added? Was it semi-recent? Asking because I was wondering if we should gate this param with the vault version as well.

Reply:
I thought it was older, but apparently it was added in october - i'll gate it behind the 1.19 flag.